.changeset/sour-mirrors-wave.md

```diff
@@ -0,0 +1,61 @@
+---
+"@fluidframework/presence": minor
+---
+---
+"section": other
+---
```

Comment:
```suggestion
---
"@fluidframework/presence": minor
"__section": other
---
```

Changeset format has changed recently, so we need to do a little manual fix up here.